README.md

```diff
@@ -1,5 +1,7 @@
 # FFI::Aspell
 
+[![Build Status](https://travis-ci.org/YorickPeterse/ffi-aspell.svg?branch=master)](https://travis-ci.org/YorickPeterse/ffi-aspell)
+
```

Comment:
Yes, I don't want all these kind of badges in the README; they're not useful to end users.